### PR TITLE
Add package comments

### LINKED ISSUE #15: Document package level documentation
Document package level documentation.

Make sure to explain which file with a valid comment takes precedence. I.e., the first one file alphabetically.

Fixed by #17

### PR DESCRIPTION
**Pull request by mfridman**
Fixes #15

### CODE REVIEW

File: docs/bsr/documentation.md [hunk 1]
```diff
@@ -34,6 +34,25 @@ You can quickly navigate from the docs to the Protobuf file by clicking the file
...
+The following is an example as displayed in the screenshot.
+
+```proto
+syntax = "proto3";
+
+// Package weather describes the weather service and 
+// all corresponding RPCs.
+// 
+// Important, for historical reasons this package 
+// must use Celsius as the default unit.
```
Comment (johanbrandhorst) [inline docs/bsr/documentation.md:52]: This reads a bit strange, should it be
```suggestion
// Note, for historical reasons this package 
// must use Celsius as the default unit.
```

Or 
```suggestion
// Important: for historical reasons this package 
// must use Celsius as the default unit.
```

File: docs/bsr/documentation.md [hunk 2]
```diff
@@ -34,6 +34,25 @@ You can quickly navigate from the docs to the Protobuf file by clicking the file
...
+package weather;
```
Comment (johanbrandhorst) [inline docs/bsr/documentation.md:53]: Should we use our best practice nameing guide for the packages here?
```suggestion
package weather.v1;
```

Comment (mfridman) [reply docs/bsr/documentation.md:53]: Yep, good catch. Will update

**Comment by mfridman** [comment]
For brevity, updated screenshot in 779f377c

**Pull request merged**